arches/app/utils/decorators.py

```diff
@@ -59,6 +61,24 @@ def new_func(*args, **kwargs):
     return new_func
 
 
+def login_required(function):
+    @functools.wraps(function)
+    def wrapper(request, *args, **kwargs):
```

Comment:
@RobOatesHistoricEngland - Is this function name descriptive enough for what it does?  Wrapper seems fairly generic to me

Reply:
It's used extensively within decorators.py and is an internal throw-away name. Stuck with the convention. The real function is called login_required.